src/vte.c

```diff
@@ -268,10 +268,11 @@ static void create_vte(void)
 	vc->menu = vte_create_popup_menu();
 	g_object_ref_sink(vc->menu);
 
-	frame = gtk_frame_new(NULL);
+	viewport = gtk_viewport_new (NULL, NULL);
+	gtk_viewport_set_shadow_type(GTK_VIEWPORT(viewport), GTK_SHADOW_NONE);
```

Comment:
Because frame shows, well, frame around (notice there was frame before around the terminal - with this patch the terminal area gets 2px bigger on every side because the frame disappears).

I haven't found a way to hide the frame around GtkFrame and also didn't find other convenient GtkContainer that could be used (I think GtkScrolledWindow was useable too but it sounds even stranger).